### DISCUSSION TITLE
Portable LMN-3 Build Pictures

### DISCUSSION THREAD

**Discussion by kdr24912** [FundamentalFrequency/.github#57] [Show and tell]
Hi all,
I’d like to share my recent LMN-3 build that I completed over the weekend. As I’m sure you’ll notice, I’ve added a battery to my LMN so it can be more portable. Aside from adding the 26800mAh battery bank and removing the sides, this is a stock build. The added thickness isn't bad at all and the machine sits nicely in the lap when playing.

The LMN has been a blast to play with for the last day or so, I can’t wait to keep playing!

![20220725_182632](https://user-images.githubusercontent.com/93159810/180896464-d395bdfb-fb4c-4490-acae-6807be9b01a9.jpg)

![20220725_182647](https://user-images.githubusercontent.com/93159810/180896511-36360fcc-d4aa-467b-b879-500953022a3b.jpg)

![20220725_182802](https://user-images.githubusercontent.com/93159810/180896627-a7875b6d-8f2f-4ad4-a72f-df8cb1385192.jpg)

**Comment by stonepreston** [comment]
Very nice! looks great and nice job with the battery!

**Comment by completely-lost** [comment]
Very cool. What power supply in that? I'm not too familiar the Rpi's is there any concern on losing power mid use?

**Reply by kdr24912** [reply]
Here's a [link](https://www.amazon.com/dp/B07P5ZP943?psc=1&ref=ppx_yo2ov_dt_b_product_details) to the battery pack used. 
I haven't had any issues on losing power mid use, it's a surprisingly stable battery pack. There's a basic battery life indicator on the side that'll help you keep an eye on when to power down and charge up (or you can play while you charge!).

**Comment by benpocalypse** [comment]
Very nice build! Really sleek looking.

I've got my basic build all set up here, and am looking to make a portable build out of it once I finish designing my case. I'll be sure to share pics then as well. I hope others share theirs as well.
